imperial_coldfront_plugin/tasks.py

```diff
@@ -37,3 +37,14 @@ def wrapped(*args, **kwargs):
         return async_task(func, *args, **kwargs)
 
     return wrapped
+
+
+def run_consistency_check():
+    """Run the LDAP consistency check task."""
+    # Schedule the task to run daily
+    schedule(
+        "imperial_coldfront_plugin.tasks.check_ldap_consistency",
+        schedule_type="D",
+        repeats=-1,
+        next_run=datetime.datetime.now(),
+    )
```

Comment:
This function is never called so the task won't actually get scheduled. Instead we use a data migration to make sure it is always present in the database see the expiry notification email task as an example https://github.com/ImperialCollegeLondon/imperial_coldfront_plugin/blob/main/imperial_coldfront_plugin/migrations/0009_schedule_expiry_notification.py